ckanext/recombinant/templates/recombinant/resource_edit.html

```diff
@@ -59,6 +69,34 @@
                   {% snippet "recombinant/snippets/xls_download.html",
                     pkg=dataset, errors=errors %}
                 {% endif %}
+                {% if g.userobj.sysadmin and resource.datastore_active %}
+                  {# only let sysadmins datastore dump via UI #}
```

Comment:
this doesn't need to be limited to sysadmins, right? Any user with access can hit the dump url directly.

Reply:
@wardi yeah correct, but we wanted the UI for the recombinant screen to be limited to sysadmins because we thought that users may get confused between the CSV and Excel templates.

Reply:
Consider exposing it to normal users. I think we've had requests in the past where people wanted to download all their data and it would be nice to let them (just not in the excel template form because that encourages people to do things that would result in data loss)